docs/examples/go-ipfs-as-a-library/README.md

```diff
@@ -0,0 +1,111 @@
+# Using go-ipfs as a Library. Learn to spawn a node and add a file to the IPFS network
+
+> This tutorial is the sister of the [js-ipfs IPFS 101 tutorial](https://github.com/ipfs/js-ipfs/tree/master/examples/ipfs-101).
+
+By the end of this Tutorial, you will learn how to:
+
+- Spawn an IPFS node that runs in process (no separate daemon process)
+- How to create an IPFS Repo
+- How to add files & directories to IPFS
```

Comment:
@daviddias Good call -- I'd originally just started commenting because I was anticipating coming up with rationales behind the suggestions but then it turned out they were pretty self-evident. 😉 Changing those now.